books/free-programming-books-langs.md

```diff
@@ -1244,6 +1244,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Mastering Next.js](https://masteringnextjs.com)
 * [Next.js Crash Course 2021](https://www.youtube.com/watch?v=mTz0GXj8NN0) - Brad Traversy (Traversy Media)
 * [The Next.js Handbook](https://flaviocopes.com/page/nextjs-handbook/) - Flavio Copes (PDF, EPUB, Kindle) *(email address requested)*
+* [The official Next.js guide](https://nextjs.org/learn) - The official Next.js team
```

Comment:
Don't invent titles! Use the title given by the resource. Same with creators - this one may be a corporate author.

Reply:
Oho, my bad. There isn't really a title to the book, per se -- but what the creators usually call it is the "Offical Next.js guide" while referencing to it.

Plus, should I just remove the authors, then? It isn't really public knowledge on who's really writing them -- it's just marketed by the entire vercel team.

Reply:
we need to be able to see if the resource is a duplicate by the title and creators. I think the author here is the company name and look at the page for title - e.g. the html title

Reply:
As requested

```suggestion
* [Learn Next.js](https://nextjs.org/learn) - Vercel Inc.
```

Reply:
Done!